groups/ecorp/local.json

```diff
@@ -1,15 +1,27 @@
 {
-  "comment":"ECorp evil AS",
+  "comment": "ECorp evil AS",
```

Comment:
I think json files are not consistent on the spaces around ":". I'd be tempted to use no spaces around ':', seems to be what is mostly done here

Reply:
So I'm going to stick with the principle of not putting spaces around the colon.